deployment/terraform/templates/cloud-config.tpl

```diff
@@ -0,0 +1,18 @@
+#cloud-config
+
+runcmd:
+  - mkdir -p /opt/district-builder/user-data/
+  - touch /opt/district-builder/user-data/config_settings.py
+  - chown -R ec2-user:ec2-user /opt/district-builder
+  - touch /home/ec2-user/permissions-ready
+  - usermod -aG docker ec2-user
+  - yum update -y
+  - yum groupinstall -y 'Development Tools'
+  - yum install -y libffi-devel
+  - curl 'https://bootstrap.pypa.io/get-pip.py' -o '/tmp/get-pip.py'
+  - python /tmp/get-pip.py
+  - /usr/local/bin/pip install docker-compose==1.16.* awscli
```

Comment:
> I believe the AWS CLI is already installed on ECS AMIs.

I just launched an instance based on `ami-aff65ad2` (latest ECS AMI), and I actually don't think it is:
```bash
[ec2-user@ip-10-0-2-133 ~]$ for dir  in ${PATH//:/ }; do sudo find $dir -name "*aws*"; done
find: `/home/ec2-user/.local/bin': No such file or directory
find: `/home/ec2-user/bin': No such file or directory
```
Since both packages have to be installed anyway, I'm inclined to leave the pip install as is.

Reply:
I tested with a RF container instance and that had it installed. Its existence on those instances may have been influenced by the installation of `awslogs` via `cloud-config` `packages` though.

I still think that it would be worth installing the `aws-cli` package via `packages`, and if that works, download the Docker Compose binary via `curl`. That approach helps avoid needing `pip` as a dependency, as well as all of the build dependencies necessary.

```yaml
packages:
 - aws-cli
```

In looking through reference material about `packages`, I also ran into `users` and `groups`. I think that we could use them to manipulate `ec2-user` membership of the `docker` group.

```yaml
users:
  - default
  - name: ec2-user
    groups: docker
```

I realize that so much emphasis on funneling working approaches through `cloud-config` may feel valueless, but I think there is long-term value in trying to reuse those directives vs. rolling our own.